.github/ISSUE_TEMPLATE/meta_request.yml

```
@@ -0,0 +1,34 @@
+name: 🧠 Meta feature request
+description: Suggest improvements to the project
+labels: [meta]
```

Maybe like my other suggestion?
```suggestion
labels: [Meta request]
```

---

My comment here is as the other one